Generated: Annotations + Function/Action Overloads + Bug Fixes

iambmelt opened the pull request:
This generated sdk is the output of [MSGraph-SDK-Code-Generator | PR #29](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/29)

/cc @shiftylogic

Commit:
Adding Java impl of GET & POST helper classes for adding extra
parameters

Commit:
Merge branch 'master' into iambmelt/work

Commit:
Annotations + function/action overloads + bug fixes

Commit:
Making comment line formatting consistent across files

Commit:
newline fixes

Commit:
Removing debug info from templated files

Commit:
Newline consistency

Commit:
newline consistency

peternied commented on graphsdk/src/main/java/com/microsoft/graph/generated/BaseSubscription.java:
```diff
@@ -0,0 +1,101 @@
+
+
+	public BaseSubscription(){
+		oDataType = "microsoft.graph.subscription";
```
I know you didn't do this, but could you perform a tab cleansing pass?

iambmelt replied:
Yep - see https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/29/commits/97c329d738dac5df10c22915814688b98dee9317

Commit:
Changes per
https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/29/commits/d54c7ebc86c92b4d10d4946ef364983140efd8b1

Commit:
Changes per
https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/29/commits/97c329d738dac5df10c22915814688b98dee9317
